### ISSUE TITLE
Automatically switch to expanded mode when stepping into code

### ISSUE THREAD

**Issue by jtpio** [jupyterlab/debugger#212]
When the current mode is condensed (sidebar) and we step into:

- foreign code
- a custom module that lives in a different file
- a function that was defined in a cell that has been deleted
- or anything that is not visually open in the workspace

the debugger should switch to expanded mode and open the right file in the editors area.

**Comment by jtpio** [comment]
This might not be needed, and depends on #176.

**Comment by jtpio** [comment]
Closing in favor of https://github.com/jupyterlab/debugger/issues/176.
